app/processing/settlement/schema.js

```diff
@@ -1,9 +1,17 @@
 const Joi = require('joi')
 
+const isFirstPayment = (value, helpers) => {
+  if (value.endsWith('001')) {
+    return value
+  }
+  throw new Error('Not first payment')
+}
+
 module.exports = Joi.object({
   paymentRequestId: Joi.number().integer().required(),
-  invoiceNumber: Joi.string().required(),
+  invoiceNumber: Joi.string().custom(isFirstPayment).required(),
```

Comment:
This isn't ideal, but the `paymentRequests`  table does not save the `paymentRequestNumber` that tells us whether it's a first payment.  The ideal solution would be to use that to prevent generation of the statement for post payment adjustments.

As that's not saved the only way is taking the last three digits of the invoice and making sure they are `001`. ie PR1.

Deliberately filtering out post payment adjustments because the current setup of the service will produce invalid data.  Not just from the changes introduced in this PR, but existing usage of the `paymentRequest` data.  Everything uses the `completed` status request  which is fine for first payments, but values will be incorrect for anything else.  Needs to be a combination of `completed` and `in progress` to be correct.

Filtering them out here means no risk of us sending nonsense to the customer.